mxcube3/ui/components/Tasks/Characterisation.js

```diff
@@ -193,7 +191,7 @@ Characterisation = connect(state => {
   return {
     path: `${state.queue.rootPath}/${subdir}`,
     filename: `ref-${prefix}_${runNumber}.???`,
-    motorLimits: state.beamline.motorsLimits,
+    motorLimits: state.beamline.movables,
```

Comment:
@meguiraun is it possible to rename motorLimits to motors? Or something like that. Because in fact it's now the list of movables, not limits. The same goes for all of them. What do you think?